_assets/css/search-gov-theme.scss

```diff
@@ -141,6 +141,14 @@ main {
   width: 8rem;
 }
 
+// Code Block Styling
+
+pre.highlight {
```

Comment:
Can we add this styling for code as well? 
![Screen Shot 2022-12-09 at 10 42 50 AM](https://user-images.githubusercontent.com/9538411/206739077-f0008e9e-d932-4750-90fc-d0a713011d1b.png)

Reply:
added, this will impact this kind of inline code styling across the site

Reply:
hooray

Reply:
ack, things have gotten strange - the list at the top, as I should have expected, now has the `<code>` styling, not sure if there's a way to do that. inline it looks great. but the strange part is that the `<pre>` blocks now have top and bottom padding. any thoughts there?
![Screen Shot 2022-12-09 at 3 58 02 PM](https://user-images.githubusercontent.com/9538411/206795223-c855b7c3-2b45-4d6c-af47-e1a43c812d64.png)

Reply:
pushed some updates, once it builds it should be resolved.

not sure why you're still seeing that phantom gray box, we may have to screenshare next week to resolve this